core/src/OpenMPTarget/Kokkos_OpenMPTargetSpace.cpp

```diff
@@ -195,12 +197,20 @@ SharedAllocationRecord<Kokkos::Experimental::OpenMPTargetSpace, void>
   typedef SharedAllocationRecord<Kokkos::Experimental::OpenMPTargetSpace, void>
       RecordHost;
 
-  SharedAllocationHeader const *const head =
+  Header head;
+
+  Header const *const head_ompt =
       alloc_ptr ? Header::get_header(alloc_ptr) : (SharedAllocationHeader *)0;
+
+  if (alloc_ptr) {
+    Kokkos::Impl::DeepCopy<HostSpace, Experimental::OpenMPTargetSpace>(
+        &head, head_ompt, sizeof(SharedAllocationHeader));
+  }
+
   RecordHost *const record =
-      head ? static_cast<RecordHost *>(head->m_record) : (RecordHost *)0;
+      alloc_ptr ? static_cast<RecordHost *>(head.m_record) : (RecordHost *)0;
```

Comment:
(Comment got lost somehow)
You must clean that up.  Blocking until it is fixed.

Reply:
ok ok